### DISCUSSION TITLE
windows GUI not working. circling and not syncing

### DISCUSSION THREAD

**Discussion by hhenrry501** [Chia-Network/chia-blockchain#8170] [Support]
I am on Win 10 GUI, and it suddendly stopped syncing. I tried reinstall, but still not working properly.

\node_discovery.py:701> exception=AttributeError("'WalletPeers' object has no attribute 'connection'")>
Traceback (most recent call last):
  File "chia\server\node_discovery.py", line 704, in ensure_is_closed
  File "chia\server\node_discovery.py", line 118, in _close_common
AttributeError: 'WalletPeers' object has no attribute 'connection'
2021-08-23T21:23:55.808 harvester chia.harvester.harvester: WARNING  Not farming any plots on this harvester. Check your configuration.
2021-08-23T21:24:24.548 wallet asyncio                    : ERROR    Task exception was never retrieved
future: <Task finished name='Task-18' coro=<WalletPeers.ensure_is_closed() done, defined at chia\server\node_discovery.py:701> exception=AttributeError("'WalletPeers' object has no attribute 'connection'")>
Traceback (most recent call last):
  File "chia\server\node_discovery.py", line 704, in ensure_is_closed
  File "chia\server\node_discovery.py", line 118, in _close_common
AttributeError: 'WalletPeers' object has no attribute 'connection'

**Comment by BrandtH22** [comment]
@hhenrry501 , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
